services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -39,12 +43,24 @@ export class HookCallbackHandlerService {
     currentProjectId: TProjectId;
   }) {
     if (processName === 'kyc-unified-api') {
-      return await this.mapCallbackDataToIndividual(
+      const context = await this.mapCallbackDataToIndividual(
         data,
         workflowRuntime,
         resultDestinationPath,
         currentProjectId,
       );
+
+      const aml = data.aml as { endUserId: string; hits: unknown[] };
+
+      await this.updateEndUserWithAmlData({
+        sessionId: data.id as string,
+        amlHits: aml.hits,
+        withActiveMonitoring: workflowRuntime.config.ubosOngoingMonitoring ?? false,
```

Comment:
Only if you feel like it.
```suggestion
        withActiveMonitoring: !!workflowRuntime.config.ubosOngoingMonitoring,
```